README.md

```diff
@@ -11,14 +11,59 @@ It is based on: https://github.com/alphagov/product-page-example
 - `bundle exec middleman server`
 - `open http://localhost:4567`
 
+## Using Docker
+
+This application supports docker for release scripts.
+
+To make use of it, follow these steps:
+
+1. Build the docker image
+
+    ```
+    $ docker build -t paas-product-page .
+    ```
+
+    You may also want to install the existing image instead:
+
+    ```
+    $ docker pull governmentpaas/paas-product-page
+    ```
+
+1. Act on the image behalf
+
+    With this command, you'll share the current directory with the docker
+    container and connect into it.
+
+    ```
+    $ docker run -ti --rm -v $(pwd):/app paas-product-page ash
+    ```
+
+1. Change directory into the shared app one
+
+    ```
+    $ cd /app
```

Comment:
Minor suggestion: you can use the `-w` argument to combine this step with the previous command:
```
docker run -ti --rm -v $(pwd):/app -w app paas-product-page ash
```